docs/components/panel/_demo.scss

```diff
@@ -0,0 +1,25 @@
+.docs-panel {
```

Comment:
Please use `git mv` and rename file to `_index.scss`.

Reply:
Changes have been done, Please review